apps/design/frontend/src/create_election_button.tsx

```diff
@@ -58,6 +57,11 @@ export function CreateElectionButton(
     );
   }, [history, mutateCreateElection, orgId]);
 
+  if (!getUserFeaturesQuery.isSuccess) {
+    return null;
```

Comment:
I wonder if it's worth it to keep `FeaturesContext` around just to wrap this call. On error, it could return a top-level error page or default to a standard no-extras feature config. It would save us from this pattern and relieve individual consumers of the responsibility of deciding how to report errors.

Reply:
This pattern is for loading, not errors. I believe errors will still use our default error boundary.

I could see a case for using a context to factor out the loading pattern. I considered it when starting this change, but decided against it for two reasons:
1. I thought it would make it more complicated to reason about the sequence of rendering/add a different pattern of data management to think about
2. I wasn't sure if every page would want to wait for features to load before displaying anything

I don't feel super strongly about those reasons, so I'm open to arguments for changing it if you think there are compelling advantages to a context

Reply:
Ah you are right. Since we already have standardized error handling I don't think there's a strong enough reason to push for a less flexible context at this point.